docs/sources/configure-client/opentelemetry/ebpf-profiler.md

```diff
@@ -0,0 +1,50 @@
+---
+title: OpenTelemetry Profiles Support (Experimental)
+menuTitle: OpenTelemetry Profiles
```

Comment:
Are OpenTelemetry Profiles a proper noun? We use sentence case usually. 
```suggestion
menuTitle: OpenTelemetry Profiles
```

Reply:
Updated e732ce92bb942e28df452c14a9a00cc546df39a3